Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionDetailsVC.swift

```diff
@@ -0,0 +1,186 @@
+
+            siteInfoURLLabel.leadingAnchor.constraint(equalTo: siteInfoTitleLabel.leadingAnchor),
+            siteInfoURLLabel.topAnchor.constraint(equalTo: siteInfoTitleLabel.bottomAnchor, constant: 2),
+            siteInfoURLLabel.trailingAnchor.constraint(equalTo: siteInfoSection.trailingAnchor, constant: -21),
+
+            connectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: ETPMenuUX.UX.gutterDistance),
+            connectionView.topAnchor.constraint(equalTo: siteInfoSection.bottomAnchor, constant: 36),
+            connectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -ETPMenuUX.UX.gutterDistance),
+            connectionView.heightAnchor.constraint(equalToConstant: 60),
+
+            connectionImage.centerYAnchor.constraint(equalTo: connectionView.centerYAnchor),
+            connectionImage.leadingAnchor.constraint(equalTo: connectionView.leadingAnchor, constant: 20),
+            connectionImage.heightAnchor.constraint(equalToConstant: 20),
+            connectionImage.widthAnchor.constraint(equalToConstant: 20),
+
+            connectionStatusLabel.leadingAnchor.constraint(equalTo: connectionImage.trailingAnchor, constant: 28),
+//            connectionStatusLabel.topAnchor.constraint(equalTo: connectionView.topAnchor, constant: 8),
+            connectionStatusLabel.trailingAnchor.constraint(equalTo: connectionView.trailingAnchor, constant: -21),
+            connectionStatusLabel.heightAnchor.constraint(equalToConstant: 22),
+            connectionStatusLabel.centerYAnchor.constraint(equalTo: connectionView.centerYAnchor)
+
+//            connectionVerifierLabel.leadingAnchor.constraint(equalTo: connectionStatusLabel.leadingAnchor),
+//            connectionVerifierLabel.bottomAnchor.constraint(equalTo: connectionView.bottomAnchor, constant: -8),
+//            connectionVerifierLabel.trailingAnchor.constraint(equalTo: connectionView.trailingAnchor, constant: -21),
+//            connectionVerifierLabel.heightAnchor.constraint(equalToConstant: 20),
+        ])
+    }
+
+    private func updateViewDetails() {
+        siteTitleLabel.text = viewModel.topLevelDomain
+        siteInfoImage.image = viewModel.image
+        siteInfoTitleLabel.text = viewModel.title
+        siteInfoURLLabel.text = viewModel.URL
+        connectionStatusLabel.text = viewModel.connectionStatusMessage
+        connectionVerifierLabel.text = viewModel.connectionVerifier
+        connectionImage.image = viewModel.lockIcon
+    }
+
+    // MARK: - Actions
+
+    @objc func closeButtonTapped() {
+        self.dismiss(animated: true)
+    }
+}
+
+// MARK: - Themable
+extension EnhancedTrackingProtectionDetailsVC: Themeable {
+    @objc func applyTheme() {
+        if #available(iOS 13.0, *) {
+            overrideUserInterfaceStyle =  ThemeManager.instance.userInterfaceStyle
+        }
+        view.backgroundColor = UIColor.theme.etpMenu.background
```

Comment:
You can remove #available iOS13 now - we're officially supporting this onwards 🥳 . Also, this VC isn't changing when switching to dark mode until you dismiss & reopen. 

You can handle this with adding an observer on `.DisplayThemeChanged`. See tabTrayVC on how it sets up and handles those notifications.